tensorflow/python/training/initial_learning_rate_decay.py

```diff
@@ -0,0 +1,190 @@
+# Copyright 2015 Google Inc. All Rights Reserved.
```

Comment:
@vrv I see.  I somehow had the impression that the learning rate was continuously fed in and updated in the original file.  It seems like I remembered wrong.  Let me look at it again and I'll put it in the same file if I confirmed my thinking.  Thanks!